### PR TITLE
Add EIGEN_MAKE_ALIGNED_OPERATOR_NEW macros

### PR DESCRIPTION
**Pull request by mpowelson**
This appears to not fix the alignment issues, but perhaps it is a step in that direction.

### CODE REVIEW

**Comment by Levi-Armstrong** [comment]
I would compare to PR #45 to see if you found anything new.

File: tesseract_collision/include/tesseract_collision/bullet/bullet_utils.h [hunk 1]
```diff
@@ -159,7 +159,7 @@ class CollisionObjectWrapper : public btCollisionObject
   {
     m_data.push_back(t);
   }
-
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```
Comment (Levi-Armstrong) [inline tesseract_collision/include/tesseract_collision/bullet/bullet_utils.h]: Does it matter where it goes in the class/struct?

File: tesseract_collision/include/tesseract_collision/fcl/fcl_utils.h [hunk 2]
```diff
@@ -119,6 +119,7 @@ class CollisionObjectWrapper
     clone_cow->m_enabled = m_enabled;
     return clone_cow;
   }
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```
Comment (Levi-Armstrong) [inline tesseract_collision/include/tesseract_collision/fcl/fcl_utils.h]: Same as above?

File: tesseract_collision/include/tesseract_collision/bullet/bullet_utils.h (hunk 1 above)
Comment (mpowelson) [reply tesseract_collision/include/tesseract_collision/bullet/bullet_utils.h]: I don't think so. The documentation seems to imply that as long as it is public it should be fine. I can move it to the top if you prefer though.

**Pull request merged**